src/utils/os_zephyr.c

```diff
@@ -220,12 +225,22 @@ void *os_memdup(const void *src, size_t len)
 
 void *os_malloc(size_t size)
 {
+#if defined(CONFIG_WIFI_NM_WPA_SUPPLICANT_HEAP)
+	return k_heap_aligned_alloc(&wifi_nm_wpa_supplicant_mem_pool, WORD_SIZE, size, K_FOREVER);
```

Comment:
why is this aligned but the common heap isn't? If alignment is needed both version should be aligned, no?